examples/resnet/resnet_dynamic_ptq_cpu.json

```diff
@@ -28,6 +28,28 @@
             }
         }
     },
+    "data_configs": [
+        {
+            "name": "cifar10_data_config",
+            "type": "HuggingfaceContainer",
```

Comment:
I think it might be better to introduce an empty container type? This config is huggingface container in name only and doesn't behave like a regular hf container since almost all component types are overridden. 

Same applies to the dummydatacontainer config in the phi2 example.

Reply:
I had the same thought but an empty data config would still need to return something from `load_dataset_config` to `dataloader_config`. I don't know if an empty `DataConfig` would be a good idea.

Reply:
for example the updated phi2 example is a case where the dataset passed to dataloader is completely ignored.

Reply:
perhaps in the case of the "empty" ("custom") data container, it should be the responsibility of the user to provide components that are compatible with one another and where they want to use it? Although this might just encourage users to wrap the pre-existing "dataloader_func" with this container like how the phi2 example in this PR is doing

Reply:
Commented above.